Gds/src/fprime_gds/common/logger/data_logger.py

```diff
@@ -56,7 +56,6 @@ def send(self, data, dest):
 
         Arguments:
             data {bin} -- binary data packet
-            dest {string} -- where the data will be sent by the server
```

Comment:
dest is still a parameter to this function. Its documentation should be put back.